### PR TITLE
Enable SDK linking for public iOS apps and add fix to allow publishin…

### PR DESCRIPTION
**Pull request by PavelRPavlov**
…g to Apple store.

### CODE REVIEW

File: TagIt/tagit/tagit.iOS/tagit.iOS.csproj [hunk 1]
```diff
@@ -40,6 +40,7 @@
     </MtouchUseLlvm>
     <MtouchEnableBitcode>
     </MtouchEnableBitcode>
```
Comment (GeorgeDarakchiev) [inline TagIt/tagit/tagit.iOS/tagit.iOS.csproj:42]: Missing `<MtouchLink>SdkOnly</MtouchLink>`

File: TagIt/tagit/tagit.iOS/tagit.iOS.csproj [hunk 2]
```diff
@@ -67,6 +68,8 @@
     <ConsolePause>false</ConsolePause>
     <CodesignKey>iPhone Developer</CodesignKey>
     <CodesignEntitlements>Entitlements.plist</CodesignEntitlements>
```
Comment (GeorgeDarakchiev) [inline TagIt/tagit/tagit.iOS/tagit.iOS.csproj:70]: Missing `<MtouchLink>SdkOnly</MtouchLink>`

File: TagIt/tagit/tagit.iOS/tagit.iOS.csproj [hunk 3]
```diff
@@ -92,6 +95,7 @@
     <CodesignProvision>iOS Distribution TelerikTagit</CodesignProvision>
     <CodesignKey>iPhone Distribution</CodesignKey>
     <CodesignEntitlements>Entitlements.plist</CodesignEntitlements>
```
Comment (GeorgeDarakchiev) [inline TagIt/tagit/tagit.iOS/tagit.iOS.csproj:97]: Missing `<MtouchLink>SdkOnly</MtouchLink>`

File: ToDo/TodoApp/TodoApp.iOS/TodoApp.iOS.csproj [hunk 4]
```diff
@@ -79,7 +79,7 @@
...
+    <MtouchExtraArgs>--linkskip=Microsoft.EntityFrameworkCore --linkskip=Microsoft.EntityFrameworkCore.Relational --linkskip=Microsoft.EntityFrameworkCore.Sqlite --linkskip=Remotion.Linq --linkskip=Microsoft.Data.Sqlite.Core --linkskip=Microsoft.EntityFrameworkCore.Sqlite.Core --linkskip=Microsoft.Extensions.DependencyInjection.Abstractions --linkskip=Microsoft.Extensions.DependencyInjection --linkskip=Microsoft.Extensions.Logging.Abstractions --linkskip=Microsoft.Extensions.Logging --linkskip=Microsoft.Extensions.Caching.Abstractions --linkskip=Microsoft.Extensions.Caching.Memory --linkskip=Microsoft.Extensions.Options --linkskip=Microsoft.Extensions.Primitives --linkskip=System.Reflection --linkskip=System.Reflection.Primitives --linkskip=System.Linq --linkskip=System.Linq.Expressions --linkskip=System.Linq.Queryable --optimize=experimental-xforms-product-type</MtouchExtraArgs>
```
Comment (GeorgeDarakchiev) [inline ToDo/TodoApp/TodoApp.iOS/TodoApp.iOS.csproj:82]: `--optimize=experimental-xforms-product-type` should be added to "Tests|iPhoneSimulator" and "Tests|iPhone " configuration as well as we want to build the application for tests the same as the one for store.

File: ToDo/TodoApp/TodoApp.iOS/TodoApp.iOS.csproj [hunk 5]
```diff
@@ -106,7 +106,7 @@
     <CodesignProvision>Automatic:AppStore</CodesignProvision>
     <CodesignKey>iPhone Distribution</CodesignKey>
     <CodesignEntitlements>Entitlements.plist</CodesignEntitlements>
```
Comment (GeorgeDarakchiev) [inline ToDo/TodoApp/TodoApp.iOS/TodoApp.iOS.csproj:108]: Missing `<MtouchLink>SdkOnly</MtouchLink>`

File: ToDo/TodoApp/TodoApp.iOS/TodoApp.iOS.csproj (hunk 4 above)
Comment (GeorgeDarakchiev) [inline ToDo/TodoApp/TodoApp.iOS/TodoApp.iOS.csproj:82]: Missing `<MtouchLink>SdkOnly</MtouchLink>`

**Review by GeorgeDarakchiev — CHANGES_REQUESTED** [review]
I have added few comments. Also there are some places that I have added that "<MtouchLink>SdkOnly</MtouchLink>" is missing, it seems like this is the default value and there is no need to be set, but it is strange that is set in some places and not in others, my preferences would be to set it on each place, but removing it from the places where it is set works as well as long as it is consistent along the apps.

**Comment by PavelRPavlov** [comment]
I have decided to go with the default value of the MtouchLink and only add the MtouchExtraArgs where needed. This is the AppStore build configuration for the Todo, TagIt and ERP apps and the Release build configuration of the CRM app.

**Review by GeorgeDarakchiev — APPROVED** [review]

**Pull request merged**